### ISSUE TITLE
Some questions about train CT3D on three classes on kitti dataset?

### ISSUE THREAD

**Issue by czy-0326** [hlsheng1/CT3D#4]
I  train CT3D on kitti dataset three classes with 2 2080Ti GPUs,the bachsize is 2 per gpu. But the car mAP on val set is 79.38(11 recall position） and 83.48（40 recall position).If the batchsize will effect the performance？
The cfg second_ct3d_3cat.yaml lose  num_points  and why the num_classes is 1 in Transformer module(line 138 in cfg，and the learning rate and epoch is different from the paper？If the cfg file is correct?
I can not run the test.py with the your ckpt. The .pth file becomes zip file,maybe I should change the pyorch version to 1.6.
